scripts/pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.towncrier]
 package = "sanic"
-package_dir = "."
+package_dir = ".."
```

Comment:
Was this change intentional? Not exactly versed on this particular file but didn’t really see a comment about it from the PR

Reply:
it was, sorry. @harshanarayana 's 'make release' was looking for the sanic package in the scripts/ dir, not in the root of the project, and towncrier was failing to execute.